app/controllers/vm_common.rb

```diff
@@ -1677,6 +1677,39 @@ def set_form_vars
     session[:edit] = @edit
   end
 
+  ACTION_TYPES = {
+    "advanced_settings"   => ->(amount) { n_("Advanced Setting", "Advanced Settings", amount) },
```

Comment:
Ok, @Fryguy, so if we construct the hash in a function to limit the scope of the binding captured by the lambdas, you will be happy?

Or we merge this and if you @matthewd or @tenderlove or someone else come with a better solution, I or @mzazrivec will fix the occurences of this pattern.